src/library/trackmodel.h

```diff
@@ -5,6 +5,7 @@
 #include <QLinkedList>
 #include <QItemDelegate>
 #include <QtSql>
+#include <cmath>
```

Comment:
math includes are messed up by the various compilers / libraries. 
We handle this in "util/math.h" please include this instead.